### ISSUE TITLE
Mac client disconnects on its own and displays incorrect connection status

### ISSUE THREAD

**Issue by yakovmanshin** [Jigsaw-Code/outline-client#712]
**Bug Description**
Outline on my Mac sometimes stops working for no apparent reason and with absolutely no warning. It usually happens after I wake the computer after a sleep, but not every time I do so.

Websites start asking me to check if selected location is correct, and googling “my ip” returns my ISP’s IP address, not my Outline server’s.

The client app, however, shows connection status as active.

<img width="412" alt="Outline bug 1" src="https://user-images.githubusercontent.com/4648742/70224642-ae7cca80-175e-11ea-94a9-3ad81cb9b455.png">

But the weirdest thing happens when I attempt to reconnect to my server. Once I’ve disconnected and clicked the Connect button to enable Outline again, the following window appears:

<img width="532" alt="Outline bug 2" src="https://user-images.githubusercontent.com/4648742/70224787-ea179480-175e-11ea-9b81-296016c1034a.png">

(Both buttons are actually enabled; they got disabled when I was taking the screenshot.)

**To Reproduce**
1. Wait until Outline stops working but not showing so;
2. Observe that connection seems to be active, while in fact it’s not;
3. Disconnect from the server;
4. Click Connect to connect again;
5. Observe the “Outline” Would Like to Add VPN Configurations alert;
6. Click Allow.

**Expected Behavior**
Outline does not disconnect on its own. If an issue occurs that prevents the connection from staying active any longer, the app clearly indicates so and does not show an incorrect connection status.

**Screenshots**

**Desktop:**
* OS: macOS 10.15.1
* Outline 1.3.2
* Browser: Safari 13.0.3

**Additional Context**
I’ve had this issue multiple times over the past couple of days.

**Comment by alalamav** [comment]
Hi @yakovmanshin, thank you for reporting this issue. We haven't done much testing on macOS 10.15 yet, but I will try to reproduce and get back to you.

Did you ever run into this problem on macOS 10.14?

**Comment by yakovmanshin** [comment]
@alalamav No, not once. This issue is very likely related to Catalina since its first occurrence took place about the same time the macOS upgrade came out.

However, version 1.3.2 of Outline was released approximately at the same time, so it’s probably worth checking whether the v1.3.2 update could introduce the bug.

**Comment by yakovmanshin** [comment]
I stopped seeing this bug a few weeks ago, about the time when macOS 10.15.2 came out.

Not closing the issue, though, since it probably still needs some investigation.
